src/admin-views/settings/help-banner.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Settings help banner
+ *
+ * @since TBD
+ *
+ * @var bool $etp_enabled         Event Tickets Plus enabled or not.
+ * @var array $et_resource_links  Knowledgebase links for Event Tickets.
+ * @var array $etp_resource_links Knowledgebase links for Event Tickets.
+ */
+
+$help_text = $etp_enabled
+	? __( 'Thank you for using Event Tickets and Event Tickets Plus! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' )
+	: __( 'Thank you for using Event Tickets! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' );
+
+?>
+<div id="event-tickets-info">
+	<h3><?php esc_html_e( 'Getting Started With Tickets', 'event-tickets' ); ?></h3>
+	<p class="help-text"><?php esc_html_e( $help_text ); ?></p>
+
+	<div class="help-links-wrapper">
+		<div class="et-help-links">
+			<h3><?php esc_html_e( 'Beginner Resources', 'event-tickets' )  ?> </h3>
+
+			<ul class="kb-links">
+				<?php
+				foreach ( $et_resource_links as $link ) {
+					printf( '<li><a href="%s" target="_blank" rel="nofollow">%s</a></li>', esc_attr( $link['href'] ), esc_html( $link['label'] ) );
```

Comment:
```suggestion
					printf( '<li><a href="%s" target="_blank" rel="nofollow">%s</a></li>', esc_url( $link['href'] ), esc_html( $link['label'] ) );
```

Reply:
Also need the noop on the links right?

Reply:
add `noopener` and `noreferrer` to `rel` if target is `_blank`. This goes for all links in this PR that have a target of `_blank`